resources/assets/coffee/osu_common.coffee

```diff
@@ -97,6 +97,16 @@
   isMobile: -> ! window.matchMedia('(min-width: 840px)').matches
 
 
+  # mobile safari zooms in on focus of input boxes with font-size < 16px, this works around that
+  focus: (el) =>
+    return $(el).focus() if !osu.isIos
+
+    currentFontSize = $(el).css('font-size');
```

Comment:
should read from `style.fontSize` so it doesn't inline css' `font-size` when restoring.

And what's with trailing `;`.